File: 3.rb
amount = gets.to_i #ввод количества чисел
for i in (1..amount) 
    number = gets.to_i #ввод чисел
    count = 0
    while number != 0 
        if (number % 10) % 2 == 0 #проверка на чётность цифр
            count += 1 #подсчёт количества
        end
        number /= 10
    end
    print("Result: ", count, "\n")
end

File: 3.py
amount = int(input()) #ввод количества чисел
for i in range(0, amount):
    number = int(input()) #ввод чисел
    count = 0
    while number != 0:
        if (number % 10) % 2 == 0: #проверка на чётность каждой цифры
            count += 1 #подсчёт количества
        number //= 10
    print("Result: ", count)